### PR TITLE
Add to sandbox in case the card as been created into not ready

### PR DESCRIPTION
**Pull request by NeOMakinG**

### CODE REVIEW

**Comment by MatShir** [comment]
🚀

File: src/services/changeColumn.js [hunk 1]
```diff
@@ -56,6 +56,8 @@ module.exports.changeColumn = async (githubClient, issue, projectId, value) => {
     return false;
   }
 
+  console.log(fieldDatas, 'prout');
```
Comment (matks) [inline src/services/changeColumn.js:59]: Nope XD

**Review by matks — COMMENTED** [review]
Code looks valid except one line XD

File: src/services/changeColumn.js (hunk 1 above)
Comment (NeOMakinG) [reply src/services/changeColumn.js]: Again... 😆

**Review by matthieu-rolland — APPROVED** [review]

**Pull request merged**